syntax = "proto3";

package fraud_detection;

// Define the FraudDetection service
service FraudDetection {
    rpc CheckFraud (FraudRequest) returns (FraudResponse);
}

// Define the request message
message FraudRequest {
    int32 order_id = 1;
}

// Define the response message
message FraudResponse {
    bool is_fraud = 1;
}